monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -237,13 +237,36 @@ object Rx {
 }
 
 /** A smart variable that can be set manually. */
-class Var[A](initialValue: Option[A], register: Var[A] => Cancelable) extends Rx[A] {
+class Var[A](
+  initialValue: Option[A], register: Var[A] => Cancelable,
```

Comment:
I have to admit I haven't been using scalafmt on any project so far; is that what I should be using?